src/constraintsolver/ConstraintSolver.java

```diff
@@ -85,6 +85,14 @@ public InferenceSolution solve(Map<String, String> configuration, Collection<Slo
         return solution;
     }
 
+    /**
+     * sanitize the configuration of solver
```

Comment:
Make these full sentences: start with a capital letter and end with a full stop.